server/scsynth/CMakeLists.txt

```diff
@@ -231,7 +231,6 @@ elseif(AUDIOAPI STREQUAL coreaudio)
 	target_link_libraries(libscsynth "-framework CoreAudio")
 endif()
 
-target_link_libraries(libscsynth boost_system_lib boost_filesystem_lib)
```

Comment:
sclang and supernova still link with `boost_system_lib`, is that not needed here? (just noticing the difference, I don't know about the functionality etc)